### PR TITLE
Added FAQ sections for Lovelace

### PR DESCRIPTION
**Pull request by ciotlosm**
**Description:**
Added FAQ sections for Lovelace

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/lovelace/index.markdown [hunk 1]
```diff
@@ -111,3 +111,14 @@ It is possible to add your own custom cards to show up in the Lovelace UI. For m
 ## Current limitations
 
 This is the very very early version aimed at gathering feedback. Discussion and suggestions are welcome in the [ui-schema repository](https://github.com/home-assistant/ui-schema) and in the [chat](/join-chat/) in #lovelace.
+
+## FAQ
```
Comment (sjabby) [inline source/lovelace/index.markdown]: Make it a linkable title

Comment (ciotlosm) [reply source/lovelace/index.markdown]: It should be using https://www.home-assistant.io/lovelace/#faq considering https://www.home-assistant.io/lovelace/#custom-cards is also available. Is there any special function the "linkable" does?

Comment (sjabby) [reply source/lovelace/index.markdown]: linkable title will create the #link, also its in the docs standard: https://developers.home-assistant.io/docs/en/documentation_standards.html

The other titles in this file already use linkable title.

**Comment by frenck** [comment]
Is there a specific reason why this is target at the `next` branch?

**Comment by ciotlosm** [comment]
@frenck for some reason from CLI I can't use "current", like it's a reserved word.

**Comment by frenck** [comment]
@ciotlosm From CLI? What CLI?

**Comment by ciotlosm** [comment]
```
ciotlosm@local:~/git/development/home-assistant.github.io$ git branch
  add_auto_mode
  fix_docs_links
* next
ciotlosm@local:~/git/development/home-assistant.github.io$ git checkout current
error: pathspec 'current' did not match any file(s) known to git.
ciotlosm@local:~/git/development/home-assistant.github.io$ git branch -a | grep current
  remotes/origin/current
  remotes/upstream/current
```

**Comment by frenck** [comment]
@ciotlosm, you'll have to specify the remote branch to track.
E.g., `git checkout -b current origin/current`

**Pull request closed without merging**